src/muse/timeslices.py

```diff
@@ -596,4 +596,15 @@ def represent_hours(
     return convert_timeslice(DataArray([nhours]), timeslices).squeeze()
 
 
+def drop_timeslice(data: DataArray):
```

Comment:
Ideally, add output type:
```suggestion
def drop_timeslice(data: DataArray) -> DataArray:
```